fossunited/api/emailing.py

```diff
@@ -74,23 +90,30 @@ def add_to_email_group(email_group: str, email: str):
 
 
 @frappe.whitelist()
-def create_newsletter_campaign(data: dict, event: str = None, chapter: str = None):
+def create_newsletter_campaign(
+    data: dict,
+    reference_document: str = None,
+    document_type: str = EVENT,
+    chapter: str = None,
+):
     """
     Create a newsletter document linked to the particular event / chapter
 
     Args:
-        data: data to be set in the doctype
-        event: event id
-        chapter: chapter id
+        data(dict): data to be set in the doctype
+        reference_document(str): if of the reference document of type `document_type`
+        document_type(str): type of reference document linked. default: 'FOSS Chapter Event'
+        chapter(str): id of chapter it is linked to
     """
-    _event = event
+    _reference_document = reference_document
     _chapter = chapter
```

Comment:
unrelated but out of curiosity - why do you use the `_x` convention for some variable names and not others?

Reply:
No particular reason tbh